letsdrill.rb

```diff
@@ -1,13 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  #Put your code here!
+  case integer
+  when (60..69)
+  	grade = "D"
+  when (70..79)
+  	grade = "C"
+  when (80..89)
+  	grade = "B"
+  when (90 ..100)
+  	grade = "A"
+  else 
+  	grade = "F"
+  end
+  grade
 end
```

Comment:
I'm using sublime as my text editor and have it set to use two spaces as an indentation, but github is interpreting is otherwise. I added this two my sublime config and I think it is working now. 

"translate_tabs_to_spaces": true